index.html

```diff
@@ -16,7 +16,7 @@
 
 <body>
   <!-- Navigation bar by Priyanka-->
-  <nav class="navbar navbar-expand-sm bg-light justify-content-center sticky-top">
+  <nav class="navbar navbar-expand-sm bg-light justify-content-center md-sticky-top">
```

Comment:
I think this should be sticky-md-top, not md-sticky-top?

Reply:
Yes you are correct